driverconfig/config.yaml

```diff
@@ -156,6 +156,12 @@ drivers:
         registrar: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.3.0
   - name: powerstore
     configVersions:
+      - configVersion: v2.3.0
+        useDefaults: true
+        supportedVersions:
+          - version: v122
+          - version: v123
+          - version: v124
       - configVersion: v2.2.0
         useDefaults: true
         supportedVersions:
```

Comment:
Please remove the older versions of powerstore in config.yaml... v2.0.0 and older